client/state/login/selectors.js

```diff
@@ -250,16 +243,15 @@ export const getRequestSocialAccountError = ( state ) =>
  * @returns {?boolean}         Boolean describing social account linking status
  */
 export const getSocialAccountIsLinking = ( state ) =>
-	get( state, 'login.socialAccountLink.isLinking', null );
+	state.login.socialAccountLink?.isLinking ?? null;
```

Comment:
Both optional chaining and the fallback can be removed. Always a valid object and a valid boolean, respectively.

Reply:
Done.